pkg/sql/parser/sql.y

```diff
@@ -6206,14 +6177,11 @@ c_expr:
 
 // Productions that can be followed by a postfix operator.
 //
```

Comment:
I don't quite understand why you needed to introduce all of these new names. Was it not enough to just leave the RHS of these in the various rules?